tests/apollo/test_skvbc_fast_path.py

```diff
@@ -87,11 +87,17 @@ async def test_fast_to_slow_path_transition(self, bft_network, tracker):
 
         Finally the decorator verifies the KV execution.
         """
-        await bft_network.do_key_exchange()
         bft_network.start_all_replicas()
 
-        await bft_network.wait_for_fast_path_to_be_prevalent(
-            run_ops=lambda: tracker.run_concurrent_ops(num_ops=20, write_weight=1), threshold=20)
```

Comment:
hmm, I think there's some issue with your rebase - you seem to have overridden the recent changes by Kostas?

Reply:
mmm, maybe, anyway I didn't plan to merge this PR.
I'll create a new one and merge my code to recent code